manifests/base/jupyter-rocm-minimal-notebook-imagestream.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/blob/main/rocm"
+    opendatahub.io/notebook-image-name: "ROCm"
+    opendatahub.io/notebook-image-desc: "Jupyter ROCm notebook image for ODH notebooks."
+    opendatahub.io/notebook-image-order: "10"
+    opendatahub.io/recommended-accelerators: '["amd.com/gpu"]'
```

Comment:
Is this the correct accelerator?
I see that on upstream we don't specify any https://github.com/opendatahub-io/notebooks/blob/main/manifests/base/jupyter-rocm-minimal-notebook-imagestream.yaml#L18 
I guess we will need to update the imagestreams also there